lib/bolt/core/src/dep/nomad/gen.rs

```diff
@@ -726,3 +729,50 @@ async fn nomad_docker_io_auth(ctx: &ProjectContext) -> Result<serde_json::Value>
 		Ok(json!(null))
 	}
 }
+
+async fn nomad_loki_plugin_config(
+	ctx: &ProjectContext,
+	svc_ctx: &ServiceContext,
+) -> Result<serde_json::Value> {
+	if let Some(logging) = &ctx.ns().logging {
+		let _endpoint = match &logging.provider {
+			LoggingProvider::Loki { endpoint } => endpoint,
+		};
+
+		// Create log labels
+		let labels = [
+			("ns", ctx.ns_id()),
+			("service", &format!("rivet-{}", svc_ctx.name())),
+			("node", "${node.unique.name}"),
+			("alloc", "${NOMAD_ALLOC_ID}"),
+			("dc", "${NOMAD_DC}"),
+		]
+		.iter()
+		.map(|(k, v)| format!("{k}={v}"))
+		.collect::<Vec<_>>()
+		.join(",");
+
+		// Remove default log labels
+		let relabel_config = json!([
+			{
+				"action": "labeldrop",
+				"regex": "^(host|filename)$",
+			},
+		]);
+
+		Ok(json!({
+			"type": "loki",
+			"config": [{
+				// TODO: Automate proxy system to remove hardcoded URL
```

Comment:
Can you elaborate on what's needed here?

Do you mean adding a Consul service and using the Consul address?

Reply:
I'm assuming this was talking about injecting the `endpoint` in to the Traefik config. Looks like that's already done.

a3ac090

Reply:
I don't even remember adding that TODO, and the language seems like something you'd write. Maybe I copied a chunk of code from somewhere and forgot to change it?